cargo-shuttle/src/lib.rs

```diff
@@ -427,21 +437,150 @@ impl Shuttle {
             args.path.clone()
         };
 
-        // 4. Ask for the framework
+        // 4. Ask for the template
         let template = match git_template {
             Some(git_template) => git_template,
             None => {
-                println!(
-                    "Shuttle works with a range of web frameworks. Which one do you want to use?"
-                );
-                let frameworks = args::InitTemplateArg::iter().collect::<Vec<_>>();
-                let index = FuzzySelect::with_theme(&theme)
-                    .with_prompt("Framework")
-                    .items(&frameworks)
-                    .default(0)
-                    .interact()?;
-                println!();
-                frameworks[index].template()
+                // /// Can be used during testing
+                // async fn get_schema() -> Result<TemplatesSchema> {
+                //     Ok(toml::from_str(include_str!(
+                //         "../../examples/templates.toml"
+                //     ))?)
+                // }
```

Comment:
Looks like you forgot to remove this.

Reply:
I intend to leave this here for anyone to use at will